src/lib.rs

```diff
@@ -21,12 +21,8 @@
 
 
 #![feature(unsafe_destructor)]
-#![feature(default_type_params)]
 #![feature(unboxed_closures)]
-#![feature(globs)]
-#![feature(macro_rules)]
 #![feature(slicing_syntax)]
-#![feature(associated_types)]
 #![feature(old_orphan_check)]
```

Comment:
I believe that's because that feature controls some other things: https://github.com/rust-lang/rust/pull/20568#issuecomment-68813298.